docs/docs/administration/email-notification.md

```diff
@@ -0,0 +1,67 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Email Notifications
+
+Immich supports the option to send notifications via email, this option is supported for:
+
+- Creating a new user .
+- A user is added to a shared album (send email to the newly shared user).
+- A shared album is added with new assets (send an email to all shared users beside the one that is adding assets).
+
+The easiest way to set up a mail server for notifications is through Google, you may want to avoid using Gmail for privacy reasons or as part of your degoogle plan, you can use any SMTP server that can support login using name and password.
```

Comment:
Maybe this will be better in the Guide section. In this section we just describe that is the Email notification feature, how can we set it up, what are the parameter...etc

Reply:
Maybe remove this section?

Reply:
I don't think we should remove this section completely, as this is a scenario many (if not most) users will run into. I agree with Alex in that we should move this section to a guide though

Reply:
So what sections would we like to leave on this page?